components/elm/src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -869,7 +869,8 @@ subroutine CanopyFluxes(bounds,  num_nolakeurbanp, filter_nolakeurbanp, &
             p = filterp(f)
             c = veg_pp%column(p)
             if(.not.veg_pp%is_fates(p)) then
-               if (veg_pp%itype(p) == nsoybean .or. veg_pp%itype(p) == nsoybeanirrig) then
+               ! soybean (crop with N fixation)
+               if (crop(veg_pp%itype(p)) >= 1 .and. nfixer(veg_pp%itype(p)) == 1) then
```

Comment:
I'm a little worried about this change. This code is specific to soybean since it is drought tolerant. I worry if using an nfixer flag any future crops that are classified as nfixers would inadvertently be included in this call. Currently we don't have any other nfixers, but I don't know if that will change.

Reply:
Good point. This suggests that what we want is a drought-tolerant flag for crops, or better yet a change to the photosynthesis parameters controlling btran. Let's put this on our list for testing and changes in a future PR.